git/bin/update.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+pushd $1
+git pull origin master
```

Comment:
Maybe you should checkout master first before pulling.

Reply:
yeah, that's actually a very good point! will adapt!

Reply:
done